src/octopeer-github/main/Database/EventObject/KeystrokeEvent.ts

```diff
@@ -2,13 +2,19 @@
  * Created by Maarten on 26-05-2016.
  */
 
+const KeystrokeType = {
```

Comment:
TSLint complains that (variable) KeystrokeType hasn't been used.
I'm not sure how this could have passed?